src/main/java/com/zendesk/maxwell/schema/columndef/EnumColumnDef.java

```diff
@@ -29,7 +29,7 @@ private String asString(Object value) {
 		Integer i = (Integer) value;
 
 		if ( i == 0 )
-			return null;
+			return "";
```

Comment:
this emulates what mysql does when you pass it an invalid enum; it's stored internally as a "0" but when SELECTed it appears as the empty string.